src/containers/Tenant/Query/QueryEditorControls/QueryEditorControls.tsx

```diff
@@ -59,12 +79,19 @@ export const QueryEditorControls = ({
 }: QueryEditorControlsProps) => {
     const querySelectorMenuItems = useMemo(() => {
         const titles = enableAdditionalQueryModes
-            ? QueryModeSelectorTitles
-            : OldQueryModeSelectorTitles;
+            ? QueryModeSelectorInfo
+            : OldQueryModeSelectorInfo;
 
-        return Object.entries(titles).map(([mode, title]) => {
+        return Object.entries(titles).map(([mode, {title, description}]) => {
             return {
-                text: title,
+                text: (
+                    <LabelWithPopover
+                        className={b('item-with-popup')}
+                        contentClassName={b('popup')}
```

Comment:
I know, that you've taken these classes from `Settings` but it's actually popover, not popup)